templates/site/js/load-node-content.js

```diff
@@ -15,12 +15,12 @@ function binNodes(registry, binSize) {
     return bins
 }
 
-function buildNodeDescription(name, data, node_index, row_id_suffix) {
+function buildNodeDescription(nodeName, data, node_index, row_id_suffix) {
     let node_id_suffix = `${row_id_suffix}-${node_index}`;
     const node_template = document.getElementById("link-description").cloneNode(true)
-    let href = `node.html?node=${name}`;
+    let href = `node.html?node=${nodeName}`;
     let img_src;
-    let img_alt = `${name} icon`;
+    let img_alt = `${nodeName} icon`;
```

Comment:
image alts are user facing so this should be the node name not the id:

```javascript
let img_alt = `${data.name} icon`;
```

Reply:
Changed to the node's name